wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -24,16 +26,40 @@ namespace :wagn do
 
     # Note Cardio based, but some use cases need this in app dir.  Solution?
     ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
-     
+
     puts "creating"
     Rake::Task['db:create'].invoke
 
     puts "loading schema"
     Rake::Task['db:schema:load'].invoke
-    
+
+    Rake::Task['wagn:seed'].invoke
+  end
+
+  desc "clear and load fixtures with existing tables"
+  task :create do
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
+
+    Rake::Task['wagn:clear'].invoke
+
+    Rake::Task['wagn:seed'].invoke
+  end
+
+  desc "empty the card tables"
+  task :clear => :environment do
+    conn = ActiveRecord::Base.connection
+
+    puts "delete all data in bootstrap tables"
+    WAGN_BOOTSTRAP_TABLES.each do |table|
+      conn.delete "delete from #{table}"
+    end
+  end
+
+  desc "Load bootstrap data into database"
+  task :seed do
```

Comment:
YIKES!  If we're talking about CLI to wagn task, it should be direct mapping, eg:

seed -> seed
reseed -> reseed
load -> load....

AFAIC, you can rename rake tasks with abandon, but please don't change the basic installation process for app decks.

But first, can you please explain how this passes Semaphore?  How does the database ever get created??